group_vars/customer/patient0/development/configs/edxapp/cms/settings.yml.j2

```diff
@@ -2,14 +2,16 @@
 
 PLATFORM_NAME: "{{ project_display_name }}"
 
+SITE_NAME: "{{ project_display_name }}"
```

Comment:
The SITE_NAME is used as the domain to build absolute urls included in emails...

You should remove all mentions to it in Arnold as it is now configured with a good default in fun-platform: 
https://github.com/openfun/fun-platform/blob/master/config/lms/docker_run_production.py#L193